manage.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python

```

Comment:
Delete the database from git control, its not needed , if you have initial data that should be shared with the project, implement it as [fixtures](https://docs.djangoproject.com/en/2.0/howto/initial-data/)

Reply:
I put the migrations folder in .gitignore to take the database off git control

Reply:
the migrations should be in git , cause it something that you have to share with your collaborators to do the same change you do on database.. only binary database should not be there

Reply:
Okey I will add it again and remove database binary